### PR TITLE
0.109.0

### PR DESCRIPTION
**Pull request by AaronO**

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+- fix(node/events): make `EventEmitter.call` compatible with es5 (#1315)
+- chore(node/url): add tests for `fileURLToPath` and `pathToFileURL` (#1309)
+- perf(crypto): reduce one mircotask (#1307)
+- docs: change CI rule (#1305)
+- chore: deprecate ws module (#1306)
+- chore(node/_util): remove `any` from `_util_callbackify.ts` (#1303)
+- chore(node/querystring): add test case for escape (#1304)
+- feat(std/node/stream): add partial support for `stream/web` (#1297)
+- fix(node/_tools): use denoflate to decompress Node test folder (#1299)
+- fix(std/node/module): treat .mjs files as ESM (#1301)
```
Comment (bartlomieju) [inline Releases.md:16]: Sort alphabetically and remove chore/test

Comment (AaronO) [reply Releases.md]: Done

**Review by bartlomieju — APPROVED** [review]

**Review by bartlomieju — CHANGES_REQUESTED** [review]
Actually it's not listing https://github.com/denoland/deno_std/pull/1292

**Comment by AaronO** [comment]
> Actually it's not listing #1292

It was merged to main as a chore:
> chore(examples): update chat server example (#1292)

**Review by bartlomieju — APPROVED** [review]

**Comment by kt3k** [comment]
> It was merged to main as a chore:

Right. Should have renamed the PR title as well.

**Pull request merged**